addons/udes_priorities/models/priority.py

```diff
@@ -87,42 +106,37 @@ def _check_for_duplicate_names_for_same_pick_type(self):
             raise UserError("Name must be unique for the picking types it is shown on")
 
     @api.multi
-    def _number_of_oustanding_pickings(self):
-        Picking = self.env["stock.picking"]
-        self.ensure_one()
-        return Picking.search_count(
-            [("priority", "=", self.reference), ("state", "not in", ("cancel", "done")),]
-        )
-
-    @api.multi
-    def _check_for_oustanding_pickings(self):
+    def _check_for_outstanding_pickings(self):
 
         install_mode = self.env.context.get("install_mode", False)
 
         if install_mode:
             return
 
-        priorities_with_outstanding_picks = self.browse()
-        for priority in self:
-            if priority._number_of_oustanding_pickings() > 0:
-                priorities_with_outstanding_picks |= priority
+        privileged_user = (
+            self.env.uid == SUPERUSER_ID
+            or self.env.user.has_group("udes_security.group_debug_user")
+            or self.env.user.has_group("udes_security.group_trusted_user")
+        )
+
+        priorities_with_outstanding_picks = len(
```

Comment:
len is not needed, empty recordset evaluates to false, and non empty recordset evaluates to true. So we don't need the > 0 in line 126.